pandera/checks.py

```diff
@@ -1022,3 +1021,43 @@ def _str_length(series: pd.Series) -> pd.Series:
                 f"str_length({min_value}, {max_value})",
             ),
         )
+
+    @classmethod
+    @st.register_check_strategy(st.isin_strategy)
+    @register_check_statistics(["allowed_values"])
+    def mustbein(cls, allowed_values: Iterable, strip_white_space_series: bool = False, **kwargs) -> "Check":
+        """Ensure series contains all values of within a list. The series must contain all values from the list.
+
+        :param allowed_values: The set of values that must be present. May be any iterable.
+        :param strip_white_space_series: Whether to strip white space from the series (only applies if str).
+        :param kwargs: key-word arguments passed into the `Check` initializer.
+
+        :returns: :class:`Check` object
+
+        .. note::
+            It is checked whether all elements of a list passed in as allowed values are present within a
+            :class:`pandas.Series`.
+        """
+        # Turn allowed_values into a pd.Series.
+        try:
+            allowed_values = pd.Series(allowed_values)
+        except TypeError as exc:
+            raise ValueError(
+                f"Argument allowed_values must be iterable. Got {allowed_values}"
+            ) from exc
+
+        def _mustbein(series: pd.Series) -> pd.Series:
+            """Comparison function for check"""
+            if not strip_white_space_series:
```

Comment:
see comment about removing this option from the check

Reply:
Thanks so much for all of the comments! I tried to resolve (and run the pre-commit hooks and tests). I'm submitting a new PR.